sass/app.scss

```diff
@@ -1,7 +1,184 @@
 /* apply a natural box layout model to all elements, but allowing compnents to change */
+@import "bourbon";
+@import "neat";
+@import "normalize";
+
+$skater_black: #292b29;
+$skater_light_black: adjust-hue($skater_black, 25deg);
+$skater_red: #b51d2a;
+$skater_white: #e9e7e7;
+$shadow_grey: #a7abaa;
+$fa-font-path: "//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts";
+@import "font-awesome";
+
+/* GLOBAL STYLING */
 html {
+  position: relative;
   box-sizing: border-box;
+  background: url(img/bg.jpg);
 }
 *, *::before, *::after {
   box-sizing: inherit;
 }
+
+a {
+  text-decoration: inherit;
+  color: $skater_white;
+}
+
+.container {
+  max-width: 1040px;
+  margin: auto;
+
+}
+
+/** HERO STYLING **/
+.hero {
+  display: flex;
```

Comment:
The hero here doesn't need to be a flex container. With images, it looks like flexing is stretching the display ratio.

`width: 100%` on the `.hero-bg` alone should get this fixed. Also note that on `.hero-bg` you still have the `top` set even though you are statically positioned.

After fixing those two things you can set a `max-height` to a percentage so that the logo shrinks as the screen gets smaller.